safegraph_patterns/tests/params.json.template

```diff
@@ -8,5 +8,5 @@
   "aws_secret_access_key": "",
   "aws_default_region": "",
   "aws_endpoint": "",
-  "test": "True"
+  "sync": false
```

Comment:
worth adding a test where sync is true and testing by mocking and checking with `assert_called*`?

Reply:
In the near future, yes; as part of this PR, I'm ambivalent.  Was looking to revamp the data fetching package-wide in #366, but I can put something temporary in here now if you think it would be bad to leave untested now.

Reply:
I don't feel too strongly, it's a single `if sync` statement so fine to leave it for now, and the data fetching thing makes sense as well.